fcrepo-http-api/src/test/java/org/fcrepo/http/api/services/HttpRdfServiceTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.http.api.services;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import org.fcrepo.http.commons.api.rdf.HttpIdentifierConverter;
+import org.fcrepo.kernel.api.models.FedoraResource;
+import org.fcrepo.kernel.api.RdfStream;
+import javax.ws.rs.core.MediaType;
+import org.apache.jena.rdf.model.Model;
+import org.junit.Ignore;
+import org.junit.Test;
+import java.io.InputStream;
+import java.io.ByteArrayInputStream;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * Unit tests for HttpRdfService
+ * @author bseeger
+ * @since 2019-11-08
+ */
+@RunWith(MockitoJUnitRunner.Silent.class)
+public class HttpRdfServiceTest {
+
+
+    @Mock
+    HttpIdentifierConverter idTranslator;
+
+    @Mock
+    FedoraResource resource;
+
+    @InjectMocks
+    private HttpRdfService httpRdfService;
+
+    private final String resourceUri = "www.example.com/fedora/rest/resource1";
+    private final String systemId = "info:fedora/resource1";
+    private final String rdfString = "@prefix dc: <http://purl.org/dc/elements/1.1/> . \n" +
```

Comment:
you probably already know this, but you'll want to test some triples that have objects that are URIs, including some that are in domain and some that aren't.

Also, all the `www.example.com` things should begin with `http://` so that are valid absolute uris, otherwise they will be evaluated as relative

Reply:
Yes - that's the second pass I want to make: ensure that the triples that need to get changed actually change.  I'll add the `http://` to URIs.

Reply:
I have tests for this now.